packages/payment/integration-tests/__tests__/services/payment-module/index.spec.ts

```diff
@@ -9,14 +12,134 @@ import {
   createPayments,
 } from "../../../__fixtures__"
 import { getInitModuleConfig } from "../../../utils/get-init-module-config"
-import { initModules } from "medusa-test-utils"
-import { Modules } from "@medusajs/modules-sdk"
 
 jest.setTimeout(30000)
 
 describe("Payment Module Service", () => {
   let service: IPaymentModuleService
 
+  describe("Payment Module Flow", () => {
+    let repositoryManager: SqlEntityManager
+    let shutdownFunc: () => Promise<void>
+
+    beforeAll(async () => {
+      const initModulesConfig = getInitModuleConfig()
+
+      const { medusaApp, shutdown } = await initModules(initModulesConfig)
+
+      service = medusaApp.modules[Modules.PAYMENT]
+
+      shutdownFunc = shutdown
+    })
+
+    afterAll(async () => {
+      await shutdownFunc()
+    })
+
+    beforeEach(async () => {
+      await MikroOrmWrapper.setupDatabase()
+      repositoryManager = await MikroOrmWrapper.forkManager()
+
+      service = await initialize({
+        database: {
+          clientUrl: DB_URL,
+          schema: process.env.MEDUSA_PAYMNET_DB_SCHEMA,
+        },
+      })
+
+      if (service.__hooks?.onApplicationStart) {
+        await service.__hooks.onApplicationStart()
+      }
```

Comment:
todo: we should not do that and only use the initModules above, can you update that please. You can look what we have done in the region fif you need too